extension/golang/environment.go

```diff
@@ -286,7 +286,7 @@ func (thisEnvironment *Environment) dispatchEventWithResource(schemaID string, h
 		return err
 	}
 
-	defer thisEnvironment.updateResourceInContext(context, resource)
+	//defer thisEnvironment.updateResourceInContext(context, resource)
```

Comment:
It panicked during unit-tests. Please add it again and try to fix implementation.